packages/app/src/components/Home.js

```diff
@@ -136,6 +136,21 @@ const textSection = css`
   }
 `;
 
+const nextDates = [
```

Comment:
Could be great to link meetup date OR maybe just do each 2 weeks than a simple array of dates

Reply:
👍